modules/trafficgateBidAdapter.js

```diff
@@ -0,0 +1,92 @@
+import { getWindowLocation, deepAccess } from '../src/utils.js';
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+import { BANNER, NATIVE, VIDEO } from '../src/mediaTypes.js';
+
+const BIDDER_CODE = 'trafficgate';
+const URL = '//[HOST].bc-plugin.com/?c=o&m=multi';
```

Comment:
Can you please hardcode this to https?

Reply:
Hello. What if site use http only? It will error 'https request over http'?

Reply:
In my experience, sending request to secure endpoints from http webpages doesn't cause any issue. You can test it out locally if you want, but I ran the [hello world page](https://github.com/prebid/Prebid.js/blob/master/integrationExamples/gpt/hello_world.html) with a local web server running http, and it did made the request to the secure appnexus endpoint.

Also, I see that the endpoint doesn't have a fixed host. Is there any specific reason to have a "variable" host name?